ui/src/lib/access.worker.js

```diff
@@ -19,7 +27,7 @@ export async function generateAccess({
 	satelliteUrl = "12tRQrMTWUWwzwGh18i7Fqs67kmdhH9t6aToeiwbo5mfS2rUmo@us2.storj.io:7777",
 	passphrase = ""
 }) {
-	await wasmInit;
+	await instantiateStreaming;
```

Comment:
`await wasmInit` waited until the code has been loaded. `instantiateStreaming` is a function not a promise, so this won't have the same effect. You should `await response` instead.